dojo/fixtures/test_type.json

```diff
@@ -1,723 +1,730 @@
 [
```

Comment:
This file will be removed soon.
It's better to let it as-is. It is not required anymore to make modifications in it.

Reply:
Done

Reply:
How is this going to be handled in the future?
Will the test types be generated automatically?

Reply:
yes.
this part is dynamic since few release now. better to let it die

Reply:
I could not spot a SecureCodeBox Test Type when using HTTP GET in the swagger docs. Any advice?

Reply:
@damiencarol

Reply:
I assumed to find it here. 

![image](https://user-images.githubusercontent.com/28805778/117765918-07dc0980-b22f-11eb-8d78-99164174be17.png)